local_settings.py.example

```diff
@@ -105,6 +105,56 @@ RELATE_EDITABLE_INST_ID_BEFORE_VERIFICATION = True
 
 # }}}
 
+
+# {{{ user full_name format
+
+# RELATE's default fullname format is "'%s %s' % (first_name, last_name)",
+# you can override it by supply a customized method/fuction, with 
+# "firstname" and "lastname" as its paramaters, and return a string. 
+
+# For example, you can define it like this:
+
+#<code>
+#   def my_fullname_format(firstname, lastname)
+#         return "%s%s" % (last_name, first_name)
+#</code>
+
+# and then uncomment the following line and enable it with:
+
+#RELATE_USER_FULL_NAME_FORMAT_METHOD = my_fullname_format
+
+# You can also import it from your custom module.
+
+# }}}
+
+# {{{ system email appelation priority
+
+# RELATE's default email appelation of the receiver is a ordered list:
+# ["first_name", "email", "username"], when first_name is not None
+# (e.g, first_name = "Foo"), the email will be opened
+# by "Dear Foo,". If first_name is None, then email will be used
+# as appelation, so on and so force.
+
+# you can override the appelation priority by supply a customized list,
+# available list named RELATE_EMAIL_APPELATION_PRIORITY_LIST. The available
+# elements include first_name, last_name, get_full_name, email and
+# username.
+
+#RELATE_EMAIL_APPELATION_PRIORITY_LIST = ["first_name", "email", "username", "full_name"]
+
+# }}}
+
+# {{{ 
+
+# When displayed in lists, user realname is diplayed by two fields,
+# last_name and first_name, so that the bpth fields can be ordered.
+# You can set the following configuration to True if you want to
+# display name using full_name format.
+
+#RELATE_DISPLAY_USER_FULLNAME_IN_LIST = True
```

Comment:
I add this configuration because gradebook-participants, e.g., [this page](https://relate.cs.illinois.edu/course/CS210/grading/participants/), which is 404 to me, are displaying user names using last_name and first_name, I don't know if you intended to display in this way so that you can sort user by their last_name or first_name. But for some user like me, I just want to sort user by their full_name since it can be searched. So I add ``RELATE_DISPLAY_USER_FULLNAME_IN_LIST``, so that it display full_name instead of two separated fields. I think you are confused because of the naming.